animation_nodes/sockets/object.py

```diff
@@ -59,6 +59,16 @@ def createObject(self):
             data = bpy.data.curves.new("Curve", "CURVE")
             data.dimensions = "3D"
             data.fill_mode = "FULL"
+        if type == "GPENCIL":
+            data = bpy.data.grease_pencils.new("ANGPencil")
+            layer = data.layers.new("AN-Layer", set_active = True)
+            frame = layer.frames.new(0)
+            stroke = frame.strokes.new()
+            stroke.display_mode = "3DSPACE"
+            stroke.material_index = 0
+            material = bpy.data.materials.new('AN-Material')
+            bpy.data.materials.create_gpencil_data(material)
+            data.materials.append(material)
```

Comment:
I think all of this should be removed. We should leave everything at default.

Reply:
I'll make the changes.

Reply:
It is done.